stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -143,19 +155,20 @@ constructor(
             return field
         }
 
-    @Volatile
-    private var connectionId: String = ""
+    private val _connectionId: MutableStateFlow<String> = MutableStateFlow("")
```

Comment:
This `connectionId` is never invalidated, shouldn't we invalidate it whenever the connection is not alive anymore?
Imagine the case where a user login but after sometime he loose network connection, so the `connectionId` is not valid anymore but we keep using it

Reply:
DONE